src/contentscript/overlay/root/components/Profile/ModalLogin/index.tsx

```diff
@@ -142,11 +142,7 @@ export const ModalLogin = ({
                           <img className={styles.walletsIcon} src={walletIcons[x.type]} alt="" />
                         ) : null}
 
-                        {x.type === 'dapplets' ? (
-                          <img className={styles.walletsIcon} src={walletIcons[x.type]} alt="" />
-                        ) : (
-                          <img className={styles.walletsIcon} src={x.meta.icon} alt="" />
-                        )}
+                        <img className={styles.walletsIcon} src={walletIcons[x.type]} alt="" />
```

Comment:
Replace the icon URL here:
https://github.com/dapplets/dapplet-extension/blob/203843f93a8093632a524efad3ec164b10d56018/src/background/wallets/ethereum/metamask.ts#L126

As here:
https://github.com/dapplets/dapplet-extension/blob/203843f93a8093632a524efad3ec164b10d56018/src/background/wallets/near/near/index.ts#L110

And rethink this condition. Seems that an error is here